### PR TITLE
Update `update_datasets` script

### PR DESCRIPTION
**Pull request by liamocn**
## Related issues

#155 

## Checklist

- [ ] **DO** Unit tests pass.
- [ ] **DO** If new feature, created unit test.

## Purpose

This PR ensure the `update_datasets` script skips directories that are not in either the `/projects` or `/investigator` directories. An ERROR message was being thrown every time this script ran because of searching for DATS files where they don't exist.

## Current behaviour

All subdirectories are searched for a DATS.json file including those where none are expected

## New behaviour

Only the `/projects` and `/investigators` directories are searched 
 
#### Does this introduce a major change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: app/cli.py [hunk 1]
```diff
@@ -209,6 +210,11 @@ def _update_datasets(app):
                 print("\033[0m")
                 continue
 
+        # The folowing relates to the DATS.json files of the projects. Skip directories that aren't projects or investigators.
+        patterns = [app.config['DATA_PATH'] + '/conp-dataset/projects/*', app.config['DATA_PATH'] + '/conp-dataset/investigators/*']
```
Comment (xlecours) [inline app/cli.py:214]: you can remove investigator. It doesn't exists anymore. 
If we ever add it back we will change this line.

**Review by xlecours — CHANGES_REQUESTED** [review]

**Review by cmadjar — CHANGES_REQUESTED** [review]
@liamocn this is failing Travis. If you could also integrate Xavier's comment at the same time, that would be awesome. Thanks!

File: app/cli.py [hunk 2]
```diff
@@ -209,6 +210,12 @@ def _update_datasets(app):
                 print("\033[0m")
                 continue
 
+        # The folowing relates to the DATS.json files of the projects.
```
Comment (cmadjar) [inline app/cli.py:213]:
```suggestion
        # The following relates to the DATS.json files of the projects directory in the conp-dataset repo.
```

**Review by cmadjar — CHANGES_REQUESTED** [review]
one tiny change on the comment. Otherwise, looks good!!

**Review by cmadjar — APPROVED** [review]
LGTM

**Pull request merged**